core/src/main/java/com/klaytn/caver/utils/AccountKeyPublicUtils.java

```diff
@@ -51,8 +51,8 @@ public static AccountKeyPublic decompressKey(String compressedPublicKey) {
         compEnc[0] = (byte)(yBit ? 0x03 : 0x02);
         ECPoint ecPoint = CURVE.getCurve().decodePoint(compEnc);
         return AccountKeyPublic.create(
-                Numeric.prependHexPrefix(ecPoint.getAffineXCoord().toString()),
-                Numeric.prependHexPrefix(ecPoint.getAffineYCoord().toString())
+                Numeric.toHexStringWithPrefixZeroPadded(ecPoint.getAffineXCoord().toBigInteger(),64),
```

Comment:
```suggestion
                Numeric.toHexStringWithPrefixZeroPadded(ecPoint.getAffineXCoord().toBigInteger(), 64),
```
I recommend you to follow Java code convention. 'A blank space should appear after commas in argument lists' (https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html)